src/main/java/com/google/sps/servlets/GoogleMapkeyServlet.java

```diff
@@ -0,0 +1,21 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+/** 
+ * Servlet sends Google maps API key as a String.
+ * */
+@WebServlet("/google-map-key")
+public class GoogleMapkeyServlet extends HttpServlet {
```

Comment:
Nit: `GoogleMapKeyServlet` with a capital K reads better is I think proper casing for this

Reply:
it does. I thought it was capitalized thx.